### ISSUE TITLE
GPU memory increase during training.

### ISSUE THREAD

**Issue by qsunyuan** [wusize/ovdet#17]
Excellent work!

I run ov-coco benchmark using the following command on 3090 x 8 with cuda 11.3 and torch1.12.1:

```
export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
bash tools/dist_train.sh configs/baron/ov_coco/baron_kd_faster_rcnn_r50_fpn_syncbn_90kx2.py 8
```

but the GPU memory keep increase during training. Any suggestion?

**Comment by wusize** [comment]
No worries about the memory increase as the RPN are trained to recall more regions of interest. However, half the learning rate and double the training iteration number as you use 8 GPUs instead of 16.
